TI-reports/2025/2025-Q2-BEST-WG.md

```diff
@@ -0,0 +1,224 @@
+
+### Purpose
+
+* Provide baseline security education for developers.
+
+### Current Status
+
+* As of 2025-03-25 our 2025 enrollments in all variations was 1,993. Enrollments in LFD121 was 1,649, LFD-121 (Japanese) was 32, LFD104x (edX first part) was 296, and LFD104x-JP (edX first part Japanese) was 16\.
+
+### Up Next
+
+* Continue to maintain content.
+
+## Questions/Issues for the TAC
+
+* Are there other ways we can get the word out for people to take the course and/or integrate it into organizations?
+
+## EDU.SIG
+
+### Purpose
+
+* Deliver Baseline Secure Software Development Education and Certification to All. Provide access to open and widely available education materials to all learners. Materials will be maximally accessible and easy to consume for all learners.
+
+### Current Status
+
+* The "[Security for Software Development Managers](https://openssf.org/blog/2025/03/06/new-free-course-security-for-software-development-managers-lfd125/)" course is now available online. As of 2025-03-25 has 281 enrollments.  
+* The final changes and scripting for LFEL1001, "[Understanding the EU CRA](https://docs.google.com/presentation/d/1j3OlNz2k5rk9KRD8ZZz8xvsM_hyxqOioK4UUkJTWee8/edit)" (aka “CRA 101”), have been completed. We are trying to get the free course released mid-April  
+* An improved format of "[A Brief Introduction to Developing Secure Software](https://docs.google.com/presentation/d/12b7Wm6KRp9kd1oV3QVrJpiCWVDexxF8doYZOFOfLy2Y/edit)" (\< 1 hour summary) is available.  
+* "What's in the SOSS" podcast on education has been recorded and will be released soon. Eight more guests are scheduled, with expected releases in February and March.
+
+### Up Next
+
+* Currently pressing to get release of LFEL 1001 \- please help us spread the word in mid-April\!
+
+## Concise Guides
+
+### Purpose
+
+* Artifacts that consolidate BEST practices in OSS software development and management techniques
+
+### Current Status
+
+* Guides under active development:   
+  * [C/C++ Compiler Option Hardening](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C%2B%2B.md)
+  * [Python Secure Coding Guide](https://github.com/ossf/wg-best-practices-os-developers/tree/main/docs/Secure-Coding-Guide-for-Python)
+* Completed Guides (under maintenance)
+  * [Concise Guide for Developing More Secure Software](https://best.openssf.org/Concise-Guide-for-Developing-More-Secure-Software)
+  * [Concise Guide for Evaluating Open Source Software](https://best.openssf.org/Concise-Guide-for-Evaluating-Open-Source-Software)
+  * [npm Best Practices Guide](https://github.com/ossf/package-manager-best-practices/blob/main/published/npm.md)
+  * [Source Code Management Platform Configuration Best Practices Guide](https://best.openssf.org/SCM-BestPractices/)
+
+### Up Next
+
+* Kicking off an effort to translate some of the guides to Swedish  
```

Comment:
Wow, that's pretty niche. :-)

Reply:
Granted, our friends from Sweden typically master English pretty well. But on a more serious note: this is actually based on an older [request](https://github.com/ossf/wg-best-practices-os-developers/issues/464). We recently introduced a lightweight process for [translations](https://github.com/ossf/wg-best-practices-os-developers/blob/main/docs/translations.md) as there is also work ongoing to translate the LFD121 course to Japanese. Based on this process, the author(s) of the original issue intend to contribute Swedish translations.